src/sagemaker/workflow/airflow.py

```diff
@@ -134,3 +157,56 @@ def training_config(estimator, inputs=None, job_name=None):  # noqa: C901 - supp
         train_config['S3Operations'] = s3_operations
 
     return train_config
+
+
+def tuning_config(tuner, inputs, job_name=None):
+    train_config = training_config(tuner.estimator, inputs, job_name)
+
+    train_config.pop('Tags', None)
+    train_config.pop('TrainingJobName', None)
+    s3_operations = train_config.pop('S3Operations', None)
+    hyperparameters = train_config.pop('HyperParameters', None)
```

Comment:
I think it might be cleaner to generate a base config that training and tuning build on

Reply:
Add method just exporting base training config that used by both.